### DISCUSSION TITLE
能否出个中枢网关软件版, 或者docker版本, 在linux设备上直接就可以运行.

### DISCUSSION THREAD

**Discussion by zhkufish** [XiaoMi/ha_xiaomi_home#69] [Ideas]
能否出个中枢网关软件版, 或者docker版本, 在linux设备上直接就可以运行. 为什么不买中枢网关设备? 主要还是因为贵.homeassistant就已经实现了中枢网关的功能了.

**Comment by oopuuu** [comment]
怕是没睡醒，哈哈哈哈

**Comment by HoTK1** [comment]
你哪怕说是卖授权都可能性大些，而且犯不上。光卖软件还得出更多人去做售后，还不如打包硬件一起卖，和路由器、还有以后可能出的nas放一起，还能做个卖点

**Comment by micturkey** [comment]
不能什么都想要
小米也是要赚钱的，软件已经开放到HA了，硬件总不能不给他赚钱吧

**Comment by VergilGao** [comment]
中枢网关才300块钱……
都搞智能家居了不至于300块钱都要省吧。

**Comment by mywp** [comment]
建议支持全部设备不依赖中枢可以完全本地接入局域网控制更好，包括网关也能局域网接入，而不是部分设备，这样所有协议类型（WiFi/蓝牙/Zigbee等）都支持本地接入ha控制了，也解决了大量设备和超复杂处理逻辑时中枢性能不够的问题
目前的疼点应该是要么不能接入，要么接入的功能受限制

**Reply by huhuime** [reply]
本地控制令牌泄露的时候就等这一个一个重置吧，第3方可以不负责，官方可逃不掉

**Comment by hi94740** [comment]
我觉得不如把思路逆转一下，出个内置home assistant的中枢网关😆

**Comment by weiyideai520** [comment]
非常需要这个功能
